dissect/target/plugins/os/windows/network.py

```diff
@@ -0,0 +1,317 @@
+    QLLC = 68
+    FASTETHER_FX = 69
+    CHANNEL = 70
+    IEEE80211 = 71
+    IBM370PARCHAN = 72
+    ESCON = 73
+    DLSW = 74
+    ISDN_S = 75
+    ISDN_U = 76
+    LAP_D = 77
+    IPSWITCH = 78
+    RSRB = 79
+    ATM_LOGICAL = 80
+    DS0 = 81
+    DS0_BUNDLE = 82
+    BSC = 83
+    ASYNC = 84
+    CNR = 85
+    ISO88025R_DTR = 86
+    EPLRS = 87
+    ARAP = 88
+    PROP_CNLS = 89
+    HOSTPAD = 90
+    TERMPAD = 91
+    FRAMERELAY_MPI = 92
+    X213 = 93
+    ADSL = 94
+    RADSL = 95
+    SDSL = 96
+    VDSL = 97
+    ISO88025_CRFPRINT = 98
+    MYRINET = 99
+    VOICE_EM = 100
+    VOICE_FXO = 101
+    VOICE_FXS = 102
+    VOICE_ENCAP = 103
+    VOICE_OVERIP = 104
+    ATM_DXI = 105
+    ATM_FUNI = 106
+    ATM_IMA = 107
+    PPPMULTILINKBUNDLE = 108
+    IPOVER_CDLC = 109
+    IPOVER_CLAW = 110
+    STACKTOSTACK = 111
+    VIRTUALIPADDRESS = 112
+    MPC = 113
+    IPOVER_ATM = 114
+    ISO88025_FIBER = 115
+    TDLC = 116
+    GIGABITETHERNET = 117
+    HDLC = 118
+    LAP_F = 119
+    V37 = 120
+    X25_MLP = 121
+    X25_HUNTGROUP = 122
+    TRANSPHDLC = 123
+    INTERLEAVE = 124
+    FAST = 125
+    IP = 126
+    DOCSCABLE_MACLAYER = 127
+    DOCSCABLE_DOWNSTREAM = 128
+    DOCSCABLE_UPSTREAM = 129
+    A12MPPSWITCH = 130
+    TUNNEL = 131
+    COFFEE = 132
+    CES = 133
+    ATM_SUBINTERFACE = 134
+    L2_VLAN = 135
+    L3_IPVLAN = 136
+    L3_IPXVLAN = 137
+    DIGITALPOWERLINE = 138
+    MEDIAMAILOVERIP = 139
+    DTM = 140
+    DCN = 141
+    IPFORWARD = 142
+    MSDSL = 143
+    IEEE1394 = 144
+    IF_GSN = 145
+    DVBRCC_MACLAYER = 146
+    DVBRCC_DOWNSTREAM = 147
+    DVBRCC_UPSTREAM = 148
+    ATM_VIRTUAL = 149
+    MPLS_TUNNEL = 150
+    SRP = 151
+    VOICEOVERATM = 152
+    VOICEOVERFRAMERELAY = 153
+    IDSL = 154
+    COMPOSITELINK = 155
+    SS7_SIGLINK = 156
+    PROP_WIRELESS_P2P = 157
+    FR_FORWARD = 158
+    RFC1483 = 159
+    USB = 160
+    IEEE8023AD_LAG = 161
+    BGP_POLICY_ACCOUNTING = 162
+    FRF16_MFR_BUNDLE = 163
+    H323_GATEKEEPER = 164
+    H323_PROXY = 165
+    MPLS = 166
+    MF_SIGLINK = 167
+    HDSL2 = 168
+    SHDSL = 169
+    DS1_FDL = 170
+    POS = 171
+    DVB_ASI_IN = 172
+    DVB_ASI_OUT = 173
+    PLC = 174
+    NFAS = 175
+    TR008 = 176
+    GR303_RDT = 177
+    GR303_IDT = 178
+    ISUP = 179
+    PROP_DOCS_WIRELESS_MACLAYER = 180
+    PROP_DOCS_WIRELESS_DOWNSTREAM = 181
+    PROP_DOCS_WIRELESS_UPSTREAM = 182
+    HIPERLAN2 = 183
+    PROP_BWA_P2MP = 184
+    SONET_OVERHEAD_CHANNEL = 185
+    DIGITAL_WRAPPER_OVERHEAD_CHANNEL = 186
+    AAL2 = 187
+    RADIO_MAC = 188
+    ATM_RADIO = 189
+    IMT = 190
+    MVL = 191
+    REACH_DSL = 192
+    FR_DLCI_ENDPT = 193
+    ATM_VCI_ENDPT = 194
+    OPTICAL_CHANNEL = 195
+    OPTICAL_TRANSPORT = 196
+    WWANPP = 243
+    WWANPP2 = 244
+
+
+def _try_value(subkey: KeyCollection, value: str) -> str | list | None:
+    try:
+        return subkey.value(value).value
+    except (RegistryValueNotFoundError, RegistryKeyNotFoundError):
```

Comment:
```suggestion
    except RegistryValueNotFoundError:
```
I think, because you already pass in a subkey.